docs/07_how-to/36_jms.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: 'How to consume from and publish to a JMS Provider'
+sidebar_label: 'Consume and publish JMS'
+id: ht-jms
+keywords: [jms, ibm, mq, data pipeline]
+tags:
+- jms
+- ibm
+- mq
+- data pipeline
+---
+
+IBM Message Queue enables different applications to send and receive messages reliably through message queues.
```

Comment:
@ZuhaaJamani can we flip this so we lead with JMS and then say "here's an example using IBM MQ" instead of leading with IBM MQ please

Reply:
I'll do this when I do my edit later today.